ConsoleApp1ClassWork13.05/ConsoleApp1ClassWork13.05/Program.cs

```diff
@@ -0,0 +1,151 @@
+﻿//1. Add meeting - without walidation
+//0. Exit calendar
+//2. Show all meeting
+using System;
+using System.IO.Enumeration;
+using static System.Net.Mime.MediaTypeNames;
+
+Typle<string, int ,string, string>[] meetings = new Typle<string, int, string, string>[0];
+const string FileName = "meetings.csv";
+
+void SaveToFile()
+{
+
+}
+void ShowAll()
+{
+    Console.WriteLine($"Duration ", 25);
+    Console.WriteLine($"Room ", 25);
+    Console.WriteLine($"Name ", 25);
+    Console.WriteLine();
+    var fileContent= File.ReadAllLines(FileName);
+    foreach (var line in fileContent)
+    {
+        var meetingContent = line.Split(',');
+        Console.WriteLine($"{meetingContent[0], 20}"
+            +$"{meetingContent [1], 20}"
+            + $"{meetingContent[2], 20}");
+    }
+    Console.ReadLine();
+}
+ bool GetRoom (DateTime dateTime, string roomName)
```

Comment:
If method is called GetRoom, my expectation would be that it returns a room. But here it returns bool, so more appropriate method name here may be "IsRoomExists" or "IsRoomOccupied", depending on how we use this method and what meaning does it have